www/forecast.html

```diff
@@ -0,0 +1,23 @@
+<div id="forecast" class="weather-box">
+  <h4 class="title">Forecast</h4>
+  <ion-scroll direction="x" id="forecast-scroll">
+    <div id="hourly-forecast">
+      <div class="hourly-hour" ng-repeat="hour in current.hourly.data">
+        <div class="time" ng-bind="hour.time * 1000 | date:'h a'"></div>
+        <weather-icon icon="hour.icon"></weather-icon>
+        <div class="temp">{{hour.temperature | tempInt}}&deg;</div>
```

Comment:
That is dead code, made in attempt to split up the temp from forecast.io throughout the day in one expression.